sdk/synapse/Azure.Analytics.Synapse.AccessControl/README.md

```diff
@@ -59,26 +59,37 @@ The way you control access to Synapse resources is to create role assignments. A
 The Azure.Analytics.Synapse.AccessControl package supports synchronous and asynchronous APIs. The following section covers some of the most common Azure Synapse Analytics access control related tasks:
 
 ### Role assignment examples
+* [Create access control client](#create-access-control-client)
 * [Create a role assignment](#create-a-role-assignment)
 * [Retrieve a role assignment](#retrieve-a-role-assignment)
 * [List role assignments](#list-role-assignments)
 * [Delete a role assignment](#delete-a-role-assignment)
 
+### Create access control client
+
+`CreateAccessControlClient` creates a role assignment.
+
+```C# Snippet:CreateAccessControlClient
+// Replace the string below with your actual workspace url.
+string workspaceUrl = "<my-workspace-url>";
```

Comment:
I would recommend using `endpoint` here to be consistent with the parameter name and also across of our libraries